pages/k8s/.markdownlint.json

```diff
@@ -0,0 +1,3 @@
+{
+    "MD033": false
```

Comment:
TIL about markdownlint!

Reply:
yeah, that day i learnt how to get rid of the warnings i don't want to see!